zerver/lib/email_mirror.py

```diff
@@ -28,11 +32,13 @@ def redact_stream(error_message):
     return error_message
 
 def report_to_zulip(error_message):
+    # type: (str) -> None
     error_stream = Stream.objects.get(name="errors", realm__domain=settings.ADMIN_DOMAIN)
     send_zulip(error_stream, "email mirror error",
                """~~~\n%s\n~~~""" % (error_message,))
 
 def log_and_report(email_message, error_message, debug_info):
```

Comment:
`email_message` should probably be `Dict[str, text_type]` (or maybe `Dict[text_type, text_type]`, I'm not sure which is better, but I think both are okay if they don't give mypy errors).

Since `email_message.get("From")` would then be `text_type`,
```py
"Sender: %s\n%s" % (email_message.get("From"), redact_stream(error_message))
```
will also be `text_type`. But that's not very apparent. I think you should mark it as a unicode string (`u"Sender: %s\n%s"`). Similarly you should prefix `u` to `"Stream: %s\n%s"` and `"Realm: %s\n%s"`.